modules/gdpr_fields/gdpr_fields.info.yml

```diff
@@ -0,0 +1,9 @@
+name: General Data Protection Regulation (GDPR) - Fields
+description: Allow tracking and removing of user data in all fields.
+core: 8.x
+type: module
+package: General Data Protection Regulation
+
+dependencies:
+  - gdpr:gdpr
+  - ctools:ctools (>=8.x-3.0-beta1)
```

Comment:
Features from gdpr:gdpr_dump are used, but the module is not set up as a dependency.

Reply:
Good catch!